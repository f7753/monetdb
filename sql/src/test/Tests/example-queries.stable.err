stderr of test 'example-queries` in directory 'src/test` itself:


# 15:50:01 >  
# 15:50:01 >  Mtimeout -timeout 480 Mserver -monetrc /var/tmp/manegold.GNU_32-d.6756/monet/.GNU_32-d/Linux2.2.14-12smp/share/Monet/monet.Mtest.conf -db Tests_src_test /var/tmp/manegold.GNU_32-d.6756/sql/src/parser/catalog.mil /var/tmp/manegold.GNU_32-d.6756/sql/src/parser/aserver.mil
# 15:50:01 >  

server listening on port 45123

# 15:50:10 >  
# 15:50:10 >  Mtimeout -timeout 210 sql_aclient  < example-queries.sql
# 15:50:10 >  


# 15:50:10 >  
# 15:50:10 >  Done.
# 15:50:10 >  

