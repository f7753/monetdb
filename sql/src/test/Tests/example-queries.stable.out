stdout of test 'example-queries` in directory 'src/test` itself:


# 15:50:01 >  
# 15:50:01 >  Mtimeout -timeout 480 Mserver -monetrc /var/tmp/manegold.GNU_32-d.6756/monet/.GNU_32-d/Linux2.2.14-12smp/share/Monet/monet.Mtest.conf -db Tests_src_test /var/tmp/manegold.GNU_32-d.6756/sql/src/parser/catalog.mil /var/tmp/manegold.GNU_32-d.6756/sql/src/parser/aserver.mil
# 15:50:01 >  

# Monet Database Server V4.3.3
# Copyright (c) 1993-2001, CWI. All rights reserved.
# compiled for i686-pc-linux-gnu/32bit; dynamically linked.
# dbfarm:/var/tmp/manegold.GNU_32-d.6756/monet/.GNU_32-d/Linux2.2.14-12smp/dbfarm
# dbname:Tests_src_test
# dbdir:/var/tmp/manegold.GNU_32-d.6756/monet/.GNU_32-d/Linux2.2.14-12smp/dbfarm/Tests_src_test

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.
[ 8 ]
13
4
6
2
6
10
0
quit;

# 15:50:10 >  
# 15:50:10 >  Mtimeout -timeout 210 sql_aclient  < example-queries.sql
# 15:50:10 >  

connecting
connected
1	"test"	0.1
5	"xtest"	0.9
9	"testx"	9.1
0	"texst"	99999.1
8	"texxst"	0.0001
1	"test"	0.1
5	"xtest"	0.9
9	"testx"	9.1
0	"texst"	99999.1
8	"texxst"	0.0001

# 15:50:10 >  
# 15:50:10 >  Done.
# 15:50:10 >  

